src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +56,31 @@
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        UserService userService = (UserService) injector.getInstance(UserService.class);
+        AuthenticationService authenticationService = new AuthenticationServiceImpl(userService);
+
+        User newRegisteredUser = null;
+        try {
+            System.out.println("=========================================");
+            newRegisteredUser = authenticationService.register("mary@mail.com", "111111");
+        } catch (RegistrationException e) {
+            throw new RegistrationException("Can't register user.",e);
+        }
+        System.out.println(newRegisteredUser);
+
+        try {
+            System.out.println("=========================================");
+            System.out.println(authenticationService.login("mary@mail.com", "111111"));
+        } catch (AuthenticationException e) {
+            throw new AuthenticationException("Can't login, "
+                    + "given password or email are incorrect",e);
```

Comment:
Similar to the registration exception, re-throwing `AuthenticationException` is redundant. Consider logging the exception or handling it in a way that provides more context to the user.

Reply:
ok, done